.github/workflows/bake_time.yml

```diff
@@ -11,10 +11,10 @@ on:
 
 jobs:
   baking_pull_request:
-    name: "Baking pull request..."
+    name: "Baking pull request ..."
```

Comment:
Bake time isn't going to count down with this change.  The cron version of this job is running expecting the name to match.

```
No check named 'Baking pull request...' for PR #222. 
```
From GitHub Action run 12771's [logs](https://github.com/opensearch-project/.github/actions/runs/10945551995/job/30390034013)

Reply:
I'll just revert it.